lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -123,14 +123,14 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     }
 
     // Metrics.
-    const metricAudits = category.auditRefs.filter(audit => audit.group === 'metrics');
     const metricAuditsEl = this.renderAuditGroup(groups.metrics);
 
     // Metric descriptions toggle.
     const toggleTmpl = this.dom.cloneTemplate('#tmpl-lh-metrics-toggle', this.templateContext);
     const toggleEl = this.dom.find('.lh-metrics-toggle', toggleTmpl);
-    metricAuditsEl.prepend(...toggleEl.childNodes);
+    metricAuditsEl.append(...toggleEl.childNodes);
```

Comment:
`prepend()` made the screen reader version of the category really confusing.

Maybe prepending was done because the code reads at first glance like the toggle needs to be prepended to come before the rendered audits, but `renderAuditGroup()` only renders the container and the group header, not the audits themselves, so the toggle *should* be the last child of `metricAuditsEl` at this point.

...or maybe the code just changed over time and this was a holdover of a different version :) Moved the `metricAudits` filter statement to try to make the order of things clearer, regardless.